### ISSUE TITLE
Data format check

### ISSUE THREAD

**Issue by ooookai** [agrignard/TARA#2]
## GPS
- `t1`, `c1`, `s`, `t2` they might be the GPS satellite signal metadata, which is useless for vis.
> t1= 17.8952, c1= 4.27996, s= 32.4499, t2= 17.8281

## TerMITes
- Some CSV files' headers are incorrect. `TerMITes/tara_test` is the only correct one.
> **Incorrect**
> DateTime,**_Accelerometer,x_**,y,z,Temperature,Lux,Humidity,Proximity,Pressure,Altitude,Roll,Pitch
> **Correct** 
> DateTime,**_Accelerometer x_**,y,z,Temperature,Lux,Humidity,Proximity,Pressure,Altitude,Roll,Pitch

**Comment by agrignard** [comment]
Yes t1,c1,s and t2 are useless for now and yes there is a missing coma after accelerometer this the raw termites data.
